### PR TITLE
docs: add firefox-stable channel documentation

### LINKED ISSUE #5993: [Feature] Firefox release-branch channel
It'd be nice to be able to test against Firefox built off the non-beta release branch.

steps:

- [x] build firefox-stable + juggler
- [x] add ability to install firefox-stable via cli
- [x] run tests for firefox-stable
- [ ] add types + documentation

### PR DESCRIPTION
**Pull request by aslushnikov**
Fixes #5993

### CODE REVIEW

**Review by dgozman — APPROVED** [review]

File: docs/src/browsers.md [hunk 1]
```diff
@@ -22,7 +22,47 @@ on when to opt into stable channels, refer to the [Google Chrome & Microsoft Edg
 ## Firefox
 
 Playwright's Firefox version matches the recent [Firefox Beta](https://www.mozilla.org/en-US/firefox/channel/desktop/)
-build.
+build. 
+
+### Firefox-Stable
+
+Playwright team maintains a Playwright's Firefox version that matches the latest Firefox Stable, aka `firefox-stable`.
```
Comment (mxschmitt) [inline docs/src/browsers.md:29]:
```suggestion
Playwright team maintains a Playwright Firefox version that matches the latest Firefox Stable, aka `firefox-stable`.
```
Ithink the 's is not needed here.

**Review by mxschmitt — APPROVED** [review]
do you want to add it to the types too?

([code pointer](https://github.com/microsoft/playwright/blob/master/docs/src/api/class-browsertype.md#option-browsertypelaunchchannel))

File: docs/src/browsers.md [hunk 2]
```diff
@@ -22,7 +22,47 @@ on when to opt into stable channels, refer to the [Google Chrome & Microsoft Edg
 ## Firefox
 
 Playwright's Firefox version matches the recent [Firefox Beta](https://www.mozilla.org/en-US/firefox/channel/desktop/)
-build.
+build. 
+
+### Firefox-Stable
+
+Playwright team maintains a Playwright Firefox version that matches the latest Firefox Stable, aka `firefox-stable`.
```
Comment (hg-pyun) [inline docs/src/browsers.md:29]:
```suggestion
Playwright team maintains a Playwright Firefox version that matches the latest Firefox Stable, a.k.a. `firefox-stable`.
```

File: docs/src/browsers.md [hunk 3]
```diff
@@ -22,7 +22,47 @@ on when to opt into stable channels, refer to the [Google Chrome & Microsoft Edg
...
+build. 
+
+### Firefox-Stable
+
+Playwright team maintains a Playwright Firefox version that matches the latest Firefox Stable, a.k.a. `firefox-stable`.
+
+Using `firefox-stable` is a 2-steps process:
+
+1. Installing `firefox-stable` with Playwright CLI.
+    ```shell
```
Comment (mxschmitt) [inline docs/src/browsers.md:34]: It is possible to add language-specific shell scripts, see here in the bottom of the file: https://github.com/microsoft/playwright/blob/7405655c26994a418dbeb93ad2a6cb87c77587cf/docs/src/cli.md

Comment (aslushnikov) [reply docs/src/browsers.md]: done!

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**